packages/dev/docs/pages/blog/building-a-combobox.mdx

```diff
@@ -0,0 +1,127 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import comboboxAccessibilityUrl from 'url:../assets/combobox-accessibility.mp4';
+import comboboxScrollingUrl from 'url:../assets/combobox-scrolling-safari.mp4';
+import comboboxVisualViewportUrl from 'url:../assets/combobox-visual-viewport.mp4';
+import comboboxExampleImageUrl from 'url:../assets/combobox-example.png';
+import screenreaderSpreadsheetImageUrl from 'url:../assets/screenreader-spreadsheet-combobox.png';
+
+import {BlogPostLayout, Video, Image} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+keywords: [combobox, accessibility, mobile, react spectrum, react, spectrum, interactions, touch]
+description: In this post we take a deeper look into some of the challenges faced when building an accessible and mobile friendly ComboBox.
+date: 2021-01-08
+author: '[Daniel Lu](https://github.com/LFDanLu)'
+---
+
+# Building a ComboBox
+
+<Image src={comboboxExampleImageUrl} alt="Screenshot of example ComboBox, select a major" style={{maxWidth: 'min(50%, 350px)', display: 'block', margin: '20px auto'}} />
+
+As of November 30th, [React Spectrum ComboBox](../react-spectrum/ComboBox.html) has been released as an alpha. For those who may be unfamiliar with a combobox, it is an input field that has a popover menu associated with it. The popover menu contains
```

Comment:
```suggestion
The [React Spectrum ComboBox](../react-spectrum/ComboBox.html) was released as an alpha on November 30th. For review of its functionality and purpose, a combobox is an input field that has a popover menu associated with it. The popover menu contains
```

Reply:
I feel like the tone change in the second sentence is a bit too formal, but I'll change the first one